### PR TITLE
Blog Post Number Four: First Wave of Bitcoin Grants

### PR DESCRIPTION
**Pull request by dergigi**
Announcement of bitcoin grants.

### CODE REVIEW

File: data/blog/bitcoin-grants-july-2023.mdx [hunk 1]
```diff
@@ -0,0 +1,265 @@
...
+
+We believe that freedom tech is what carries this smile forward, which is why we
+are delighted to announce grants for over a dozen projects in the bitcoin &
+lightning ecosystem.
+
+[^fn-hal]: Hal Finney: [_Why remailers..._](https://cypherpunks.venona.com/date/1992/11/msg00108.html) (November 1992)
+
+[^fn-assange]: Julian Assange: [_A Call to Cryptographic Arms_](https://cryptome.org/2012/12/assange-crypto-arms.htm) (October 2012)
+
+The following open-source projects were [selected](https://github.com/OpenSats/website/blob/master/docs/grant-selection.md) to by the
```
Comment (lucasmogu) [inline data/blog/bitcoin-grants-july-2023.mdx:27]: were selected by

**Review by lucasmogu — CHANGES_REQUESTED** [review]

File: data/blog/bitcoin-grants-july-2023.mdx [hunk 2]
```diff
@@ -0,0 +1,265 @@
...
+In addition to the open-source projects listed above, three
+educational initiatives were selected:
+
+- [Bitcoin Education in Nigeria](https://geyser.fund/project/bitcoineducationinnigeriawithapatajohnson)
+- [21 Ideas](https://www.21ideas.org/)
+- [CoreDev.tech](https://coredev.tech/)
+
+The first is an initiative started and led by Apata Johnson. Apata's project
+aims to educate youths on bitcoin and the opportunities it brings for the people
+living in the rural areas of Nigeria.
```
Comment (importantgabe) [inline data/blog/bitcoin-grants-july-2023.mdx:52]: I would put these descriptions underneath each of the bullet points, it makes it harder to follow when I read the bullet points then see the descriptions below and have to circle back to the bullets.

File: data/blog/bitcoin-grants-july-2023.mdx [hunk 3]
```diff
@@ -0,0 +1,265 @@
...
+
+The second, 21 Ideas, is a project that aims to
+bring quality Bitcoin education to Russian citizens. Tony and others have been
```
Comment (importantgabe) [inline data/blog/bitcoin-grants-july-2023.mdx:55]: Might be worthwhile to link to Apata and Tony's Twitter or Nostr profiles, I don't know who they are.

File: data/blog/bitcoin-grants-july-2023.mdx [hunk 4]
```diff
@@ -0,0 +1,265 @@
...
+### Payjoin Dev Kit
+
+Payjoin brings privacy to bitcoin without changing the way you're used to using
+it. Payjoin transactions look no different from normal activity on-chain, so
+they boost everyone's privacy, even those who don't payjoin, and foil chain
+surveillance.
+
+Payjoin is easy to integrate and falls back to working defaults where it isn't
+supported, but it can only take off when senders and receivers include standard
+payjoin support in their software. Payjoin Dev Kit makes it easy to integrate
```
Comment (importantgabe) [inline data/blog/bitcoin-grants-july-2023.mdx:79]: Makes it easy for wallet developers(?). As a non-developer I'm unclear on why this project matters (other than knowing that privacy is important).

File: data/blog/bitcoin-grants-july-2023.mdx [hunk 5]
```diff
@@ -0,0 +1,265 @@
...
+encompasses an LNURL-style protocol specification and a collection of
+open-source implementations of lnproxy clients and a relay.
+
+Repository: [lnproxy/lnproxy](https://github.com/lnproxy/lnproxy)  
+License: GPL 3.0 & MIT
+
+### Blixt Wallet
+
+Blixt is a non-custodial Lightning wallet for Android, iOS, and macOS with a
+focus on usability and user experience. The project's current target audience is
```
Comment (importantgabe) [inline data/blog/bitcoin-grants-july-2023.mdx:238]: I would start this paragraph off with "Blixt is a non-custodial wallet whose current audience is bitcoiners who want to give Lightning a try". I feel like that draws those potential people in more, and then you can give them details that they may not understand.

File: data/blog/bitcoin-grants-july-2023.mdx [hunk 6]
```diff
@@ -0,0 +1,265 @@
...
+
+Repository: [hsjoberg/blixt-wallet](https://github.com/hsjoberg/blixt-wallet)  
+License: MIT
+
+---
+
+All of the projects above are funded by contributions to the Bitcoin General
+Fund. Our grant programs are made possible by generous donors like you. If you
+want to help fund the Bitcoin ecosystem, please [donate to the Bitcoin General
+Fund](https://opensats.org/projects/general_fund).
```
Comment (importantgabe) [inline data/blog/bitcoin-grants-july-2023.mdx:254]: My opinion will always be that this should be at the top, before introducing the projects, planting the seed in readers' minds that they can donate to these projects. And then repeat the "soft ask" again at the bottom

**Pull request merged**